src/new-client/src/App/Trades/services.ts

```diff
@@ -0,0 +1,113 @@
+import { bind } from "@react-rxjs/core"
+import { combineLatest, Subject } from "rxjs"
+import { map, scan, startWith } from "rxjs/operators"
+import { trades$ } from "services/trades"
+import { ColField } from "./TradesGrid"
+
+type SortDirection = "ASC" | "DESC"
+
+class TableSort {
+  constructor(public direction?: SortDirection, public field?: ColField) {}
+}
+
```

Comment:
I am not quite familiar with class, just wondering if it's possible to create a new interface for it instead of a class.

Reply:
The benefit of the class over the interface is that it automatically types the objects it creates so you don't have to do typecasting in a lot of situations where you would with an object literal.  I started with an interface but then I was writing `{ direction: ..., field: ...} as TableSort` everywhere.